templates/layout/dev_error.php

```diff
@@ -12,14 +12,15 @@
  * @since         3.0.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
+use Cake\Error\Debugger;
 ?>
 <!DOCTYPE html>
 <html>
 <head>
     <?= $this->Html->charset() ?>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>
-        Error: <?= h($this->fetch('title')) ?>
+        Error: <?= Debugger::formatHtmlMessage($this->fetch('title')) ?>
```

Comment:
HTML tags in `<title>` not supported.

Reply:
Yes, I did. I failed to transpose the change I made in my app skeleton here :blush: